### ISSUE TITLE
Feature Request for Favicons

### ISSUE THREAD

**Issue by wolfsiren** [plainmade/unmark#102]
I would like to submit a feature request to see if it is possible to put the favicon image of  bookmark link into the view next to the bookmark. I think this would be a nice visual way to quickly identify bookmarks.

**Comment by cdevroe** [comment]
@wolfsiren That would be cool. Does anyone know if there is a "gravatar" for favicons? Or, perhaps we can write our own routine to 'check' to see if the URL that is bookmarked has a favicon. Favicons, in general, are in the same locations.

**Comment by williamknauss** [comment]
@wolfsiren and @cdevroe 
I found http://grabicon.com/ but personally I don't like the idea that Unmark would have to rely on a third party service. It might be easier to just script this ourselfs. Something like

--CRON JOB--
take Unmark "Mark" and strip URI down to subdomain.domain.tld
Curl subdomain.domain.tld/favicon.ico
If we find a favicon save it to a local path if not
Curl subdomain.domain.tld and look for the favicon in the html head then curl the favicon path.

While the methodalogy would be a little more complex than what I listed I personally think this would be a great feature addition to Unmark.

**Comment by cdevroe** [comment]
@williamknauss You're right. And this can be attached to some of the jobs we already have built into Unmark to get video/audio embeds.

**Comment by Kickball** [comment]
Any update on this task's progress?
